template.rb

```diff
@@ -208,6 +209,23 @@ def get_gh_file_content(filename)
 RUBY
 end
 
+# Initialize Gemfile.next for dual boot
+run "next --init"
+
+# Add dual-boot in Gemfile 
+next_gem = '''
+if next?
+  gem "rails", github: "rails/rails", branch: "main"
+else
+  gem "rails"
```

Comment:
we should have a version constraint here, if not it will start using the wrong version at some point when a new rails version gets released, we should parse the generated Gemfile to get the version constraint